ulib/axstarry/src/syscall_fs/ctype/eventfd.rs

```diff
@@ -1,8 +1,14 @@
 use alloc::sync::Arc;
-use axerrno::AxResult;
+use axerrno::{AxError, AxResult};
 use axfs::api::{FileIO, FileIOType};
 use axsync::Mutex;
+use axtask::yield_now;
 
+// https://sites.uclouvain.be/SystInfo/usr/include/sys/eventfd.h.html
+static EFD_SEMAPHORE: i32 = 1;
```

Comment:
You can use a bitflags type to centrally represent flags values, as axstarry/src/ctypes.rs does, to get a clearer idea of how these magic numbers are used